freud/environment.pyx

```diff
@@ -532,11 +532,56 @@ cdef class EnvironmentCluster(_MatchEnv):
                 global_search=False):
         r"""Determine clusters of particles with matching environments.
 
-        In general, it is recommended to specify a number of neighbors rather
-        than just a distance cutoff as part of your neighbor querying when
-        performing this computation. Using a distance cutoff alone could easily
-        lead to situations where a point doesn't match a cluster because a
-        required neighbor is just outside the cutoff.
+        An environment is defined by the bond vectors between a particle and its
```

Comment:
This extra-specific term also jumped out at me. I like vectors as discussed later: https://github.com/glotzerlab/freud/pull/980/files#r915013462

```suggestion
        An environment is defined by the vectors between a particle and its
```

Reply:
Could you explain why? I think "bond vector" is actually clearer, and it's consistent with the usage of "bond" in other places of the `Environment` module.

Reply:
The word "bond" doesn't add any new information.

"Vector connecting two points" is a common phrase.
Adding the word "bond" makes users wonder if there is something special happening between those particles.

Additionally, freud uses the word "bond" to mean "neighboring particles", so the term is redundant when "its neighbors" appears in the description.